### ISSUE TITLE
Notepad UI Redesign

### ISSUE THREAD

**Issue by HarshNarayanJha** [HarshNarayanJha/echo-client#1]
We need to do a quick UI redesign (or _first-_ design) of the notepad page. It looks horrible right now. I have done my best while also working on the functionality

![image](https://github.com/user-attachments/assets/2161edfd-cd9c-4caf-975b-e0c620858b74)

**Comment by TechGeekRahul** [comment]
hey @HarshNarayanJha  I want to work on this issue.can you assign it to me ?

**Comment by HarshNarayanJha** [comment]
Yeah, Sure, Go Ahead! The design is a bit updated from the screenshot as of now, though.
What kind of design are you thinking, since I also plan to add a full text editor instead of the text area

**Comment by HarshNarayanJha** [comment]
Hey @TechGeekRahul ! Any updates on the progress?
